ansible/eda-server
# [AAP-10796]: Rulebook rulesets doc string bug fix

### Jira Ticket: [AAP-10796](https://issues.redhat.com/browse/AAP-10796) ###

### Purpose: ###
- The swagger docs for the endpoint `/rulebooks/{id}/rulesets/` had the id parameter documented as "A unique integer value identifying this ruleset." when it should be "A unique integer value identifying this rulebook." because the parameter being passed is the rulebook id. 
- The problem is that the queryset for this endpoint needs to be set to the ruleset model rather than the rulebook in order for the filtering to work and the parameters are pulled from this information. Thus the id parameter needed to be overridden in order to have the correct string.

### Testing Instructions: ###
1. Ensure that the docs look as follows id parameter is for rulebooks but name filter is by rulesets.
<img width="1439" alt="Screenshot 2023-05-03 at 4 34 10 PM" src="https://user-images.githubusercontent.com/49072851/236044989-37d304ff-bb44-4461-8fb7-e1c85b369e5a.png">

2. Hit endpoint to list rulesets for a specified rulebook: `[GET]rulebooks/{id}/rulesets/` 

3. Ensure that filtering by name still works: `[GET]rulebooks/{id}/rulesets/?name=test`
